plugins/xlsx-worbook-sink/README.md

```diff
@@ -0,0 +1,9 @@
+# @flatfile/plugin-xlsx-workbook-sink"
+
+asdf...
+
+`npm i @flatfile/plugin-xlsx-workbook-sink`
```

Comment:
❔ What do we want to call plugins to "export/egress" data from Flatfile. I just chose "sink" b/c I'm used to Kafka but I'm open to anything